src/main/cli/cli.c

```diff
@@ -2703,6 +2707,58 @@ static void cliVtx(const char *cmdName, char *cmdline)
 
 #endif // VTX_CONTROL
 
+#ifdef USE_VTX_SMARTAUDIO
+static void cliVtxOpModel(const char *cmdName, char *cmdline)
+{
+    if (isEmpty(cmdline)) {
+        // If you change the model this value will be stale until reboot.
+        cliPrintLine(saDevice.willBootIntoPitMode ? "race" : "free");
+    } else {
+        bool freeModel = strncasecmp(cmdline, "free", 4) == 0;
+        bool pir = strncasecmp(cmdline, "pir", 3) == 0 || strncasecmp(cmdline, "race", 4) == 0;
+        bool por = strncasecmp(cmdline, "por", 3) == 0;
```

Comment:
You are right - the values are exclusive. I pulled them out like this because I thought it made the block of `if` statements below easier to read. I've updated the code now and inlined the `strncasecmp` calls, making clear that the choices are exclusive. I decided not to introduce an additional `enum` for this situation.